dispatch-service/src/main/resources/application.yml

```
@@ -81,7 +81,7 @@ info:
 
 swim:
   # s3 tags
-  dispatch-state-tag-key: SWIM_Status
+  dispatch-state-tag-key: SWIM_State
   dispatched-state-tag-value: sentToKafka
   dispatch-file-finished-tag-value: finished
   protocol-state-tag-key: CSV_Status
```

```suggestion
  protocol-state-tag-key: CSV_State
```

Would probably also change that to "State" for consistency.